ckanext/xloader/plugin.py

```diff
@@ -95,7 +119,16 @@ def notify(self, entity, operation):
         if _should_remove_unsupported_resource_from_datastore(resource_dict):
             toolkit.enqueue_job(fn=_remove_unsupported_resource_from_datastore, args=[entity.id])
 
-        if not getattr(entity, 'url_changed', False):
+        if utils.requires_successful_validation_report():
```

Comment:
This terminates merely if the flag is set. Doesn't it need to check whether validation actually happened?

Reply:
Here, in the `notify` method, we do not want to actually submit to XLoader if the new flag of `requires_validation` is set.

It would happen in the newly implemented `receive_validation_report` method from the `IPipeValidation` implement.

Reply:
That does make sense. Perhaps it would be better for the log message to say "Deferring"?